apps/src/templates/sectionProgress/standards/standardsConstants.js

```diff
@@ -1,3 +1,8 @@
+export const LessonIcons = {
```

Comment:
I wonder if we should try to reuse this function here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/progress/progressHelpers.js#L77-L101 ?

Reply:
That function outputs the icon for a level; in this case we're looking for the icon for a lesson.  However, I think you're on to something and that there's a way to share this constant from somewhere not standards specific.  I was planning of moving in that direction later for [LP-1338](https://codedotorg.atlassian.net/browse/LP-1338) in favor of getting standards out. Does that seem ok to you?